char_counter/widget.py

```diff
@@ -0,0 +1,16 @@
+
+from django import forms
+
+class CharCounterTextInput(forms.widgets.TextInput):
+	
+	def render(self, name, value, attrs=None, renderer=None):
+		attrs.update({'class': 'char-counter-text-input'})
+
+		textinput_html = super(CharCounterTextInput, self).render(name, value, attrs, renderer)
+
+		if not value:
+			value = ''
+
+		charcounter_html = f"<div class=\"counter\"><span class=\"input-counter\">{len(value)}</span> chars</div>"
```

Comment:
If you use single quotes on the outer string, you don't have to escape the double quotes inside.